packages/azure-kusto-data/src/cloudSettings.ts

```diff
@@ -18,7 +18,7 @@ export type CloudInfo = {
 class CloudSettings {
     METADATA_ENDPOINT = "/v1/rest/auth/metadata";
     defaultCloudInfo: CloudInfo = {
-        LoginEndpoint: process.env.AadAuthorityUri || "https://login.microsoftonline.com",
+        LoginEndpoint: process.env?.AadAuthorityUri || "https://login.microsoftonline.com",
```

Comment:
Can `process` itself be null/undefined?

Reply:
it seems like it can - although i can see that in both our setup and their setup it wasnt, also the real solution should be to set 
process.env ={} 
in some plugin

Reply:
lets add this ? anyway

Reply:
yeah

Reply:
i did